eng/Signing.props

```diff
@@ -8,128 +13,155 @@
+        <!-- Most runtime artifacts will be crossgenned, so sign them post-crossgen. mscorlib isn't. -->
+        <ItemsToSign Include="$(LibrariesSharedFrameworkBinArtifactsPath)mscorlib.dll" />
+
+        <!-- Sign the host. -->
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostfxr.dll" />
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostpolicy.dll" />
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/dotnet.exe" />
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/ijwhost.dll" />
+        <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/nethost.dll" />
+
+        <!-- Sign managed libraries in installer subset. -->
+        <ItemsToSign Include="$(ArtifactsBinDir)Microsoft.NET.HostModel/**/*.dll" />
+      </ItemGroup>
+
+      <!-- Sign ready-to-run binaries after crossgen is applied. -->
+      <ItemGroup Condition="'$(SignR2RBinaries)' == 'true'">
+        <ItemsToSign Include="$(CrossGenRootPath)**/*.dll" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignMsiFiles)' == 'true'">
+        <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.msi" />
+        <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.cab" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignBurnEngineFiles)' == 'true'">
+        <ItemsToSign Include="@(BundleInstallerEngineArtifact)" />
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignBurnBundleFiles)' == 'true'">
+        <!-- Sign the bundles, now that the engine is reattached. Avoid re-signing the engine. -->
+        <ItemsToSign
+          Include="@(BundleInstallerExeArtifact)"
+          Exclude="@(BundleInstallerEngineArtifact)" />
+        <!-- Note: wixstdba is internal to the engine bundle and does not get signed. -->
+      </ItemGroup>
+
+      <ItemGroup Condition="'$(SignFinalPackages)' == 'true'">
+        <DownloadedSymbolPackages Include="$(DownloadDirectory)**\*.symbols.nupkg" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.nupkg" Exclude="@(DownloadedSymbolPackages)" />
+
+        <!-- The cross OS diagnostics symbol packages need to be signed as they are the only packages
+        that have a specific version of assets that are only meant to be indexed in symbol servers.
+        Since only *symbols.nupkg get indexed, and installer doesn't produce these, we need to glob them for signing. -->
+        <ItemsToSign Include="$(DownloadDirectory)**\*CrossOsDiag*.symbols.nupkg" />
+
+        <ItemsToSign Include="$(DownloadDirectory)**\*.deb" />
+        <ItemsToSign Include="$(DownloadDirectory)**\*.rpm" />
+      </ItemGroup>
+    </When>
+    
+    <!-- When doing post build signing, we sign all artifacts we would push.
+         Symbol packages are included, but get no signature. -->
+    <When Condition="'$(PostBuildSign)' == 'true'">
+      <ItemGroup>
+        <ItemsToSignWithPaths Include="$(DownloadDirectory)**" />
```

Comment:
This needs to be conditioned with SignFinalPackages, otherwise it will end up globbing `**`

Reply:
Yeah, noticed that. I think SignFinalPackages would not be correct, as that would be set on the invocation of SignFinalPackages.proj, but the set ItemsToSign would not be present when we publish the artifacts. I'll figure something out.